build/index.js

```diff
@@ -4,11 +4,9 @@ const exec = require('@actions/exec');
 const checkCLI = require('../util/cli');
 
 const verbose = core.getBooleanInput('verbose');
-const skipVerification = core.getBooleanInput('skip_verification');
 
 checkCLI().then(() => {
   let params = ['compute', 'build', '--non-interactive'];
-  if (skipVerification) params.push('--skip-verification');
```

Comment:
what happens if the user chooses a cli version which is older than 6.x?  should we list that the action no longer supports older cli versions?